src/component/bookingItem/bookingItem.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from "react";
+import "./bookingItem.scss";
+import BookingStatus from "component/bookingStatus/bookingStatus";
+import { updateBookingStatus } from "api/adminApi";
+
+const BookingItem = ({
+  id,
+  services,
+  startDate,
+  endDate,
+  cusName,
+  phoneNum,
+  stylist,
+  status,
+  sttList,
+}) => {
```

Comment:
Why do we need to pass the status list into the children component like this? We can make it a constant variable in the Booking Status component. 

Avoid passing state through 2-3 children component if not necessary